modules/drivers/velodyne/launch/velodyne16.launch

```diff
@@ -0,0 +1,7 @@
+<cyber>
+    <module>
+        <name>gnss</name>
```

Comment:
Module name  is "gnss"?

Reply:
I referred to modules/drivers/velodyne/launch/velodyne128.launch , so velodyne128.launch should be corrected too?
 ```
<cyber>
    <module>
        <name>gnss</name>
        <dag_conf>/apollo/modules/drivers/velodyne/dag/velodyne128.dag</dag_conf>
        <process_name>gnss</process_name>
    </module>
</cyber>

```